.github/workflows/publish-artifacts.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish
```

Comment:
I think that this file should be removed

Reply:
Removed